v-next/hardhat/templates/01-node-test-runner-viem/package.json

```diff
@@ -6,12 +6,7 @@
   "type": "module",
   "devDependencies": {
     "hardhat": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-ignition": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-ignition-viem": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-keystore": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-node-test-runner": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-network-helpers": "workspace:^3.0.0-next.3",
-    "@nomicfoundation/hardhat-viem": "workspace:^3.0.0-next.3",
+    "@nomicfoundation/hardhat-toolbox-viem": "workspace:^3.0.0-next.3",
```

Comment:
We should double check that this works with the release script - that the toolbox, after running the script, is version `"workspace:^3.0.0-next.4"` the same as every other version.
We want to avoid having a different version number for the new packages.
The script to test is `node ./scripts/version-alpha.mjs`